src/candela/VisComponent/README.md

```diff
@@ -0,0 +1,55 @@
+# [candela](..#readme).VisComponent
+
+The base class for Candela visualization components.
+
+This class is intentionally very minimal, because there are only a few common features of
+all Candela components:
+
+1. Candela components perform some type of visualization that may be rendered in a web environment.
+2. Candela components are instantiated with the syntax [new VisComponent(el, options)](#new-viscomponentel-options).
```

Comment:
I think it might be good to explain that the ``options`` part is a best practice, rather than enforced by the API (i.e., that you can create components that don't take arguments, but most of the time your components *should* have an ``options`` parameter).

I can understand the argument for *not* explaining this here too, but I just am weary of JavaScript library documentation that keeps things implicit when they shouldn't be, or treats me with kid gloves about the way things "really" are, even when I should (without a good reason) do things the way the docs recommend, etc.